tests/Microsoft.Identity.Test.Unit/ManagedIdentityTests/ManagedIdentityTests.cs

```diff
@@ -806,10 +806,13 @@ public async Task ManagedIdentityCacheTestAsync()
         }
 
         [DataTestMethod]
-        [DataRow(1, false)]
-        [DataRow(2, false)]
-        [DataRow(3, true)]
-        public async Task ManagedIdentityExpiresOnTestAsync(int expiresInHours, bool refreshOnHasValue)
+        [DataRow(1, false, false)]
+        [DataRow(2, false, false)]
+        [DataRow(3, true, false)]
+        [DataRow(1, false, true)]
+        [DataRow(2, false, true)]
+        [DataRow(3, true, true)]
+        public async Task ManagedIdentityExpiresOnTestAsync(int expiresInHours, bool refreshOnHasValue, bool useIsoFormat)
```

Comment:
There is no assertion on `expiresInHours`. Can you add one? Otherwise, what's the point of having multiple input values for it?